src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form"
+      method="GET"
+      data-qa="big"
+    >
+      <input
+        class="search__bar"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try &#34;Los Angeles&#34;"
+      > <img
+          src="images/Search.svg"
+          alt=""
+          class="icon"
+        >
```

Comment:
Use correct indentations
```suggestion
      <input
        class="search__bar"
        type="text"
        data-qa="keypress"
        placeholder="Try &#34;Los Angeles&#34;"
      >
      
      <img
        src="images/Search.svg"
        alt="[put some text here]"
        class="icon"
      >
```